lib/sorbet-rails/model_plugins/active_record_assoc.rb

```diff
@@ -128,22 +140,4 @@ def populate_collection_assoc_getter_setter(assoc_module_rbi, assoc_name, reflec
       return_type: nil,
     )
   end
-
-  sig { params(reflection: T.untyped).returns(T.nilable(T::Boolean)) }
-  def assoc_should_be_untyped?(reflection)
-    polymorphic_assoc?(reflection) || !@available_classes.include?(reflection.klass.name)
```

Comment:
A test is now breaking (https://github.com/chanzuckerberg/sorbet-rails/blob/master/spec/model_rbi_formatter_spec.rb#L40-L55) that is testing certain classes not being in the `@ available_classes` set. Am I correct in assuming this test is meant to classes that no longer exist or can't be loaded for some reason?

If so, I think the test might not actually be tested what it's intending to. If the class were truly not in the codebase (e.g. `Message` is removed from the codebase but there's still a `has_many :messages` association lying around) `reflection.klass` this line would raise a `NameError`. This PR handles that case by rescuing the `NameError` but now the test isn't working because you can't "fake" a model not being available.

I updated the specs to remove that test and instead add an association to `Wizard` referencing a model that doesn't exist to test this out.

Reply:
>  Am I correct in assuming this test is meant to classes that no longer exist or can't be loaded for some reason?
Yeah, this test is for special scenarios when the class somehow is not loaded. We err to the side of being correct (T.untyped is safe) than being wrong